proto/message_contents/invitation.proto

```diff
@@ -70,3 +73,18 @@ message SealedInvitation {
     SealedInvitationV1 v1 = 1;
   }
 }
+
+enum ConsentProofPayloadVersion {
+  V1 = 1;
```

Comment:
We have a proto lint rule that forces you to have a prefix of the enum name on each value and to define an `$PREFIX_UNSPECIFIED` values as the `0`